assets/src/setup/components/options-context-provider.js

```diff
@@ -47,6 +47,42 @@ export function OptionsContextProvider( { children, optionsRestEndpoint } ) {
 		hasUnmounted.current = true;
 	}, [] );
 
+	/**
```

Comment:
I just moved this up because I keep expecting the `fetch` effect to be above the `save` effect.